client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,5 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
```

Comment:
If we made it non-breaking spaces, possibly, but it would need to be 5 and if the user isn't using a fixed-width font, it won't end up the exact same width as the other lines.